### ISSUE TITLE
Naming of cardinality accessors

### ISSUE THREAD

**Issue by simonwhitaker** [simonwhitaker/Swift-Bag#1]
**tl;dr: is it better to be consistent with other classes in the Swift ecosystem, or with implementations of the same data structure in other languages? Let's have a heated debate!**

---

I'm undecided on how to name the accessors responsible for the following operations:

- Getting the count for a given element (currently `count(for:)`)
- Getting the number of unique elements in the bag (currently `count`)
- Getting the overall number of elements in the bag (currently undefined)

The current state of affairs (as of d32a3ce) mirrors the functionality of [NSCountedSet](https://developer.apple.com/reference/foundation/nscountedset), which is admittedly the worst role model a data structure ever had. It is however the one that Swift programmers looking for a better bag are most likely to be familiar with.

Here's how a number of other implementations do it:

Library | count for element | # unique elements | total # elements
-|-|-|-
Swift-Bag (this library) | `count(of:)` | `count` | n/a
C++ ([std::multiset](http://en.cppreference.com/w/cpp/container/multiset)) | `count()` | n/a | `size()`
Java ([org.apache.commons.collections.Bag](https://commons.apache.org/proper/commons-collections/javadocs/api-2.1.1/org/apache/commons/collections/Bag.html)) | `getCount()` | n/a | `size()`
Java ([com.google.common.collect.Multiset](http://google.github.io/guava/releases/20.0/api/docs/com/google/common/collect/Multiset.html)) | `count()` | n/a | `size()`
Smalltalk ([Collection.Bag](https://www.gnu.org/software/smalltalk/manual-base/html_node/Bag.html)) | `occurrencesOf:` | n/a | `size`

So I'm tempted to do something like:

- `count` returns total number of elements (so the count of bag `{"a": 1, "b": 2}` would be **3**)
- `count(of:)` remains the same

These feel fairly intuitive to me, I don't think the overridden name is obnoxious or particularly confusing, and I don't _think_ anyone's going to miss compatibility with NSCountedSet's unorthodox API. What do you think?

**Comment by simonwhitaker** [comment]
OK, I've changed `count` to now return the total number of elements added (1e73d75)
